inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/AccountRepository.kt

```diff
@@ -93,6 +97,19 @@ internal abstract class AccountRepository {
             }
         }
 
+        /**
+         * From v7.1.0, the structure of cached data is changed.
+         * Remove old data since it will never be used.
+         */
+        override fun clearUserOldCacheStructure() {
```

Comment:
This will be called every change in user info. Is that intended?

Reply:
Yes. I think cache' contents needs to be updated (cleared and use new format) only when the app needs it (user changed).
We can also place this part at app startup event (loop through all of users' cache and clear them). What do you think?

Reply:
Yes, but is there a way to check that if old cache structure is no longer available, no need to call this method?

Reply:
Added checking if new key exists, if not, it means it is using old struct, then clears whole cache